docs-2.0/3.ngql-guide/4.job-statements.md

```diff
@@ -6,6 +6,25 @@ The long-term tasks run by the Storage Service are called jobs, such as `COMPACT
 
```

Comment:
a reminder that create a snapshot and do not run other jobs during SUBMIT JOB BALANCE DATA.